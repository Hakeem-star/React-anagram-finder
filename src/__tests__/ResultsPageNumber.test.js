import React from 'react';
import ResultsPageNumber from '../ResultsFound';
import renderer from 'react-test-renderer';

describe('<ResultsPageNumber />', () => {
  it('matches the snapshot', () => {
    var tree = renderer.create(<ResultsPageNumber />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});