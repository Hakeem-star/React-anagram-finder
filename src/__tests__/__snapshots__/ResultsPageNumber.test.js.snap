// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ResultsPageNumber /> matches the snapshot 1`] = `
<div
  className="ResultsPageNumber"
>
  <p
    className="previous"
  >
     
    «
     
  </p>
  <p>
    Showing 
     of 
  </p>
  <p
    className="next"
  >
     
    »
     
  </p>
</div>
`;
